package.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
- [ ] Great job so far 👏🏻 However I get this error message 👇🏻 when I run this application on my local machine. Kindy fix this and make sure your app runs and shows the resultant in the browser without error 👍🏻 

![18](https://user-images.githubusercontent.com/79058364/186335575-5faebe61-0596-4206-bcb9-7a13b4e84923.png)

Reply:
@BenMukebo, hello. Thank you for your comments, but it is very strange the error that appears to you. Just in case, I just cloned the repository to another folder, and then ran "npm install", and finally "npm start" without any problems. Attached a picture with the evidence. Sorry, but I really don't know how to proceed, because I can't reproduce that error. Moreover, it has never happened before.
Please reconsider. Or please, run it directly from index.html file of "dist" folder. Thank you


![image](https://user-images.githubusercontent.com/17918694/186340396-9d5f4a80-e502-44d5-9a2c-db6a64fbcf7c.png)

Reply:
Hello @cmolinan, Sorry for the late reply

Reply:
@BenMukebo,  I just asked a teammate to test my project and he showed me that he doesn't get bugs either.
I will resubmit the PR.   I trust that it will work well for the new Reviewer. Thank you anyway.

Reply:
Hi @cmolinan, Sorry for this, I think the problem comes from my computer, I will definitely check and fix the issue
Thank you 🙏🏻